eng/validation/build-arcadewithrepo.ps1

```diff
@@ -202,7 +204,15 @@ $barBuildId = ([regex]"\d+").Match($barBuildIdString).Value
 
 ## Make the changes to that branch to update Arcade - use darc
 Set-Location $(Get-Repo-Location $global:githubRepoName)
-& $darc update-dependencies --id $barBuildId --github-pat $global:githubPAT --azdev-pat $global:azdoToken --password $global:bartoken
+
+$darcOutput = & $darc update-dependencies --id $barBuildId --github-pat $global:githubPAT --azdev-pat $global:azdoToken --password $global:bartoken
+Write-Output $darcOutput
+if ($darcOutput -eq "Found no dependencies to update.")
+{
+   ## Log a warning so this isn't hidden, but it's not necessarily a problem either, just exit.
+   Write-Host "##vso[task.logissue type=warning]Darc returned no dependencies to update for this repo"
+   exit 0
```

Comment:
Let's have this return a failure. If the dependencies fail to update in this scenario, then we have an issue somewhere that needs to be fixed, and this particular pipeline will end up returning a false positive on a passing build with the latest Arcade version.

Reply:
OK cool.  I'll promote it to error too.

Reply:
Sure, also kicked off this version here; https://dnceng.visualstudio.com/internal/_build/results?buildId=971553&view=results